x-pack/legacy/plugins/uptime/public/lib/helper/__tests__/series_has_down_values.test.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { seriesHasDownValues } from '../series_has_down_values';
+
+describe('seriesHasDownValues', () => {
+  it('identifies that a series does have y-values', () => {
```

Comment:
Should read `down-values` not `y-values`

Reply:
Resolved by https://github.com/elastic/kibana/pull/38895/commits/67893e4fd392b8d99ed49ba40539b46fa2bea9a7.